### ISSUE TITLE
steal all outstanding features from Module::Manifest

### ISSUE THREAD

**Issue by karenetheridge** [Perl-Toolchain-Gang/ExtUtils-Manifest#11]
Module::Manifest was written to check existing MANIFEST files from anywhere in the filesystem -- see https://metacpan.org/pod/Module::Manifest#Comparison-to-ExtUtil::Manifest

However Module::Manifest does not handle #! directives, which therefore causes failures with everything using it downstream (e.g. Test::DistManifest).  Let's try to deprecate Module::Manifest, which means adding features for everything it does that EUM cannot.

**Comment by Leont** [comment]
Wouldn't it be possible to just reimplement EU::M on top of M::M? (or reimplement it for all I care) EU::M's API and interface are both stretched far beyond their original requirements, it's starting to look like a duct-tape castle.

**Comment by karenetheridge** [comment]
Yes, I'm really coming around to the view that a full rewrite is going to have to happen.

@kentnl is currently renovating the test suite, which is great as once we know we've got some reasonable tests in place, we can start hacking with less fear of breaking something (at least something that we were always testing before.  Untested regressions will probably happen too, so we'll need to be writing *more* tests as well.)
